### PR TITLE
Add Lab 3 problem and solution from class F

### PR DESCRIPTION
**Pull request by giovanism**
Lab03_F.md: Instructions and hints
Lab03_F.py: Solution

Lab03_F.py: Fix typos and small refactor

### CODE REVIEW

File: Lab/03/Lab03_F.py [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+
+print("""Lab 03
+From decimal to hexadecimal
+---------------------------""")
+
+# Get the decimal value
+# Initialize an empty string
+# Make one temporary variable for decimal value
+
+dec = int(input("Give a possitive integer in decimal representation: "))
```
Comment (laymonage) [inline Lab/03/Lab03_F.py]: possitive -> positive

Comment (laymonage) [inline Lab/03/Lab03_F.py]: possitive -> positive

File: Lab/03/Lab03_F.py [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+
+print("""
+From decimal to hexadecimal
+---------------------------""")
+
+# Get the hexadecimal value
+# Initialize new decimal and power variable
+
+str_hex = input(
+    "Give a possitive integer in hexadecimal representation: "
```
Comment (laymonage) [inline Lab/03/Lab03_F.py]: Same as above.

**Review by laymonage — CHANGES_REQUESTED** [review]

**Comment by giovanism** [comment]
Updated

**Pull request merged**